src/routes/routes.ts

```diff
@@ -115,6 +116,8 @@ router.get(pageURLs.ERROR, (res: Response) => {
 
 router.get(pageURLs.TOO_SOON, showTooSoonPage);
 
+router.get(pageURLs.EXTENSION_LIMIT_REACHED, showDueDateLimitExceededPage);
```

Comment:
I know there's been some discussion about 'extension' versus 'due date' limit reached. I think we've agreed on 'extension' now? If so, suggest naming that page var similar to not mix up the terminology, e.g. 'showExtensionLimitReachedPage'.

Reply:
Updated name